frameit/lib/frameit/editor.rb

```diff
@@ -347,10 +349,60 @@ def build_title_images(max_width, max_height)
           i.interline_spacing interline_spacing if interline_spacing
           i.fill fetch_config[key.to_s]['color']
         end
-        title_image.trim # remove white space
+
+        # title_image.trim # remove white space
 
         results[key] = title_image
+
+        # Trimming of each individual image will result in the loss of the common baseline between both texts.
```

Comment:
Move this line of comment to commented out `.trim` call above, also reformulate as we only work on an individual image here (as we are in a loop) (also: can it only be 2 texts?)

Reply:
Currently it are 1 (Title) or 2 (and Keyword).
Rephrased to:
# Trimming the image will result in the loss of the common baseline between the text in all images.